http_client.rst

```diff
@@ -1041,8 +1044,9 @@ following methods::
 Streaming Responses
 ~~~~~~~~~~~~~~~~~~~
 
-Call the ``stream()`` method of the HTTP client to get *chunks* of the
-response sequentially instead of waiting for the entire response::
+Call the ``stream()`` method of :class:`Symfony\\Contracts\\HttpClient\\HttpClientInterface`
```

Comment:
```suggestion
Call the :class:`Symfony\\Contracts\\HttpClient\\HttpClientInterface::stream`` method
```

Same for similar occurrences.

Reply:
Updated for this one and other occurrences 🙂